docs/docs/examples/emojivoto.md

```diff
@@ -154,25 +142,21 @@ The public facing frontend for voting uses the mesh certificate without client a
 
 :::
 
-## Voter's perspective: Verifying the ballot
+## Verifying the deployment as a user
 
-As voters, we want to verify the fairness and confidentiality of the deployment before
-deciding to vote. Regardless of the scale of our distributed deployment, Contrast only
-needs a single remote attestation step to verify the deployment. By doing remote attestation
-of the Coordinator, we transitively verify those systems the Coordinator has already attested
-or will attest in the future. Successful verification of the Coordinator means that
-we can be sure it will enforce the configured manifest.
+In different scenarios, users of an app may want to verify its security and identity before sharing data, e.g., before casting a vote. With Contrast, a user only needs a single remote-attestation step to verify the deployment - regardless of the size or scale of the deployment. Contrast is designed such that, by verifying the Coordinator, the user transitively verifies those systems the Coordinator has already verified or will verify in the future. Successful verification of the Coordinator means that
```

Comment:
Please put each sentence on its own line. This makes futures diffs much more readable.